tensorflow/python/ops/structured/structured_tensor.py

```diff
@@ -431,7 +438,8 @@ def _tensor_getitem(self, key):
 
   def __repr__(self):
     return '<StructuredTensor(fields={%s}, shape=%s)>' % (', '.join(
-        '%r' % k for k in sorted(self._fields)), self._shape)
+        '%s: %s' % (k, v) for k, v in sorted(self._fields.items())),
```

Comment:
This had recently changed to only show field names and not the values, which isn't very useful. I changed to also show values so that the pydoc examples would have them and be accurate to what is displayed.